### DISCUSSION TITLE
go-to-definition in new tab

### DISCUSSION THREAD

**Discussion by AbhinavAchha** [hrsh7th/nvim-cmp#765] [Q&A]
Is there any way to achieve go-to-definition in new buffer instead of current?

Thanks in advance!

**Comment by hrsh7th** [comment]
nvim-cmp does not have go-to-definition feature.

**Reply by AbhinavAchha** [reply]
ohh 
sorry my bad

**Reply by AbhinavAchha** [reply]
so is there any way to get that working with any neovim configuration?

Thanks

**Reply by arpangreat** [reply]
https://www.reddit.com/r/neovim/comments/j2ny8b/is_there_a_way_to_open_a_new_tab_when_using_lua/?utm_medium=android_app&utm_source=share


cdb_11 answer might help you alot :)

**Reply by AbhinavAchha** [reply]
I dont know why but i am getting this issue:-  attempt to index local 'callbacks' (a nil value)

callbacks = vim.lsp.callbacks

**Reply by smjonas** [reply]
The LSP API has changed in Neovim 0.6. You need to use `local callbacks = require("vim.lsp").callbacks` (can't confirm this right now so make sure to read the helpdocs about LSP if you want to make sure).

**Reply by AbhinavAchha** [reply]
Even after changing this I get the same error

 I am currently using nvim 0.7.0
